### ISSUE TITLE
Single Node Distributed Not Working

### ISSUE THREAD

**Issue by sanjaykr5** [pytorch/examples#445]
I have a p3.8xlarge aws ec2 instance and when I run the script it gets stuck without showing any error on this step: 
dist.init_process_group(backend=args.dist_backend, init_method=args.dist_url,
world_size=args.world_size, rank=args.rank)

**Comment by ElegantLin** [comment]
@yrsun I am puzzled that why I will use tcp when I only used one node.

Could you please tell me?

Thanks.

**Comment by anguoyang** [comment]
me too, have you solved the problem @sanjaykr5 @ElegantLin @yrsun

**Comment by ElegantLin** [comment]
Yes. After I upgrade the driver of GPU and install the latest version of Pytorch. I solved it.
